modules/installing-rhv-example-install-config-yaml.adoc

```diff
@@ -72,6 +78,13 @@ pullSecret: '{"auths": ...}'
 sshKey: ssh-ed12345 AAAA...
 ----
 
+<1> Setting this option to `false` enables preallocation of disks. The default is `true`. Setting `sparse` to `true` with `format` set to `raw` is not available for block storage domains. The 'raw' format writes the entire virtual disk to the underlying physical disk.
```

Comment:
One instance of the word "raw" in this line uses single quotes instead of back-ticks. Please update it to use back-ticks so that it renders as a code item.